include/interfaces/non_conservative/navier_stokes.h

```diff
@@ -256,6 +275,12 @@ NavierStokes<dim>::
 parse_parameters_call_back ()
 {
   NonConservativeInterface<dim,dim,dim+1, NavierStokes<dim> >::parse_parameters_call_back();
+
+  this->prm.leave_subsection();
```

Comment:
si potrebbe/dovrebbe aggiungere questo `prm.leave_subsection()` dentro `ParameterAcceptor` in  deal2lkit? @luca-heltai ?